### ISSUE TITLE
UI Bug

### ISSUE THREAD

**Issue by dfrnz** [videovillage/Lattice-Issues#55]
I'm on OSX 10.9.5 and there was a UI bug introduced in Lattice 1.5.4 (I think). On my 10.11 computer there is no issue.

When clicking different tabs parts of the interface disappear. Properties and Metadata cannot be seen at all, and the only way to see anything in the Image/Cube/Curves panel is to switch to the History tab and then resize the window. Here's pictures to better explain:

![screen shot 2016-01-15 at 11 56 15 am](https://cloud.githubusercontent.com/assets/16722694/12359777/1a00c42e-bb82-11e5-9c6d-8bf20e594e5c.png)
![screen shot 2016-01-15 at 12 02 54 pm](https://cloud.githubusercontent.com/assets/16722694/12359778/1a028b92-bb82-11e5-9b1a-ca2d2ae6d3fa.png)
![screen shot 2016-01-15 at 12 03 14 pm](https://cloud.githubusercontent.com/assets/16722694/12359779/1a03905a-bb82-11e5-9488-5d829ec843bd.png)

**Comment by gregcotten** [comment]
This is very upsetting! Does this only happen when the window is squeezed like you have it? Unfortunately I stopped using 10.9 around 2 years ago (before Lattice was released) and have no way of testing the UI.

Would it be possible for you to go to Console.app, search for Lattice, and copy/paste entries here?

**Comment by dfrnz** [comment]
No, I've only shrunk the window so the screenshot is smaller. And I'm only using 10.9 at work for compatibility reasons. I use 10.11 at home. Here is the log. Not sure if that is helpful as I'm not a coder.
```
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate: unbalanced enable/disable update.
1/15/16 12:57:09.450 PM Lattice[73015]: Backtrace (at 613358):
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  0   CoreGraphics                        0x00007fff90ef5a89 CGSBacktraceCreate + 59
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  1   CoreGraphics                        0x00007fff91278082 CGSLogBacktrace + 15
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  2   CoreGraphics                        0x00007fff90fa79c9 _ZN16CGSUpdateManager20enable_update_legacyEv + 87
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  3   CoreGraphics                        0x00007fff90fa796c CGSReenableUpdate + 35
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  4   QuickLook                           0x00007fff8b6b8181 _Z18QLTryCatchAndCrashU13block_pointerFvvE + 17
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  5   libdispatch.dylib                   0x00007fff8b0e91bb _dispatch_call_block_and_release + 12
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  6   libdispatch.dylib                   0x00007fff8b0e628d _dispatch_client_callout + 8
1/15/16 12:57:09.450 PM Lattice[73015]: CGSReenableUpdate:  7   libdispatch.dylib                   0x00007fff8b0e8673 _dispatch_queue_drain + 451
1/15/16 12:57:09.451 PM Lattice[73015]: CGSReenableUpdate:  8   libdispatch.dylib                   0x00007fff8b0e99c1 _dispatch_queue_invoke + 110
1/15/16 12:57:09.451 PM Lattice[73015]: CGSReenableUpdate:  9   libdispatch.dylib                   0x00007fff8b0e7f87 _dispatch_root_queue_drain + 75
1/15/16 12:57:09.451 PM Lattice[73015]: CGSReenableUpdate:  10  libdispatch.dylib                   0x00007fff8b0e9177 _dispatch_worker_thread2 + 40
1/15/16 12:57:09.451 PM Lattice[73015]: CGSReenableUpdate:  11  libsystem_pthread.dylib             0x00007fff90e85ef8 _pthread_wqthread + 314
1/15/16 12:57:09.451 PM Lattice[73015]: CGSReenableUpdate:  12  libsystem_pthread.dylib             0x00007fff90e88fb9 start_wqthread + 13
```

**Comment by johnpilgrim** [comment]
I'm have the same or similar issues with Lattice 1.5.5 on MacOS 10.9.4

**Comment by gregcotten** [comment]
Fixed in 1.5.6!
